api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionEmailAddress.java

```diff
@@ -0,0 +1,49 @@
+package org.pmiops.workbench.db.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "institution_email_address")
+public class DbInstitutionEmailAddress {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "institution_email_address_id")
+  private long institutionEmailAddressId;
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "institution_id")
+  private DbInstitution institution;
+
+  @Column(name = "email_address", nullable = false)
+  private String emailAddress;
+
+  public DbInstitutionEmailAddress(DbInstitution institution, String emailAddress) {
```

Comment:
nit: I'd probably make this class an immutable object and just give it a builder, then take away the set() functions. Unless we expect to update the fields separately, I'm guessing we just add & remove whole rows from this table. I wouldn't expose more degrees of freedom than you need to support.

Reply:
Is that possible with our DB paradigm?

Reply:
I removed a few sets we don't need - but yeah looks like we are obligate mutators